### PR TITLE
metrics:  add operator region size metric 

### LINKED ISSUE #4456: Opeator life cycle will adjust by region size not hard code includes expired、timeout、influence
## Enhancement Task
Every step should have individual  timeout that should depence on the size of the region and the time factor. The time factor should be adjust by config, like:
         `  timeout=min(10m,RegionSize* TimeFactor)`
          
### SubTask
- [x] #4457
- [x] #4547
- [x] #4459
- [x] #4783
- [x] #4935
- [x] region should devide more range to support that huge region should occupy more store limit.

### LINKED ISSUE #4935: add some metrics to show the region size of operator.
In past, the region scheduler may be slow, it is be  very helpful to diagnose.

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: Close #4935

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Integration test

image

Code changes

Side effects

Related changes

### Release note

```release-note
 None.
```

### CODE REVIEW

File: server/schedule/metrics.go [hunk 1]
```diff
@@ -34,6 +34,15 @@ var (
 			Buckets:   []float64{0.5, 1, 2, 4, 8, 16, 20, 40, 60, 90, 120, 180, 240, 300, 480, 600, 720, 900, 1200, 1800, 3600},
 		}, []string{"type"})
 
+	operatorSizeHist = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "pd",
+			Subsystem: "schedule",
+			Name:      "operator_size",
```
Comment (CabinfeverB) [inline server/schedule/metrics.go:41]: IMO, `operator_region_size` is bettoer

Comment (bufferflies) [reply server/schedule/metrics.go]: fix

**Review by CabinfeverB — APPROVED** [review]

File: server/schedule/metrics.go [hunk 2]
```diff
@@ -34,6 +34,15 @@ var (
 			Buckets:   []float64{0.5, 1, 2, 4, 8, 16, 20, 40, 60, 90, 120, 180, 240, 300, 480, 600, 720, 900, 1200, 1800, 3600},
 		}, []string{"type"})
 
+	operatorSizeHist = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "pd",
+			Subsystem: "schedule",
+			Name:      "operator_region_size",
+			Help:      "Bucketed histogram of operator size.",
```
Comment (rleungx) [inline server/schedule/metrics.go:42]:
```suggestion
			Help:      "Bucketed histogram of the operator region size.",
```

**Review by rleungx — APPROVED** [review]
The rest LGTM.

**Pull request merged**